src/test/java/uk/gov/hmcts/reform/sscs/model/SessionCategoryMapsTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import uk.gov.hmcts.reform.sscs.ccd.domain.BenefitCode;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Issue;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SessionCategory;
+import uk.gov.hmcts.reform.sscs.service.SessionCategoryMapService;
+
+public class SessionCategoryMapsTest {
+
+    SessionCategoryMapService sessionCategoryMaps = new SessionCategoryMapService();
+
+    @DisplayName("When valid Benefit Code and Issue Code is given to getHearingDuration the valid mapping is returned")
+    @Test
+    public void getSessionCategory() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("003", "LE", false, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.PIP_REASSESSMENT_CASE);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.LE);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_03);
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code and a secondDoctor is given to getHearingDuration"
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategorySecondDoctor() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("064", "GC", true, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.INDUSTRIAL_DEATH_BENEFIT);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.GC);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_06);
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code and no secondDoctor is given to getHearingDuration"
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategoryNoSecondDoctor() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("064", "GC", false, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.INDUSTRIAL_DEATH_BENEFIT);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.GC);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_05);
+    }

```

Comment:
This looks like copied code.  I'd be surprised if Sonar likes this!
Also as mentioned above, better to have a test per assert

Reply:
The test is for SessionCategory, so this is already doing that